document/core/appendix/changes.rst

```diff
@@ -104,17 +104,52 @@ Added instructions that modify ranges of memory or table entries [#proposal-reft
 * Active data and element segments boundaries are no longer checked at compile time but may trap instead
 
 
+.. index:: instructions, SIMD, value type, vector type
+
+Vector instructions
+...................
+
+Added vector type and instructions that manipulate multiple numeric values in parallel (also known as *SIMD*, single instruction multiple data) [#proposal-vectype]_
```

Comment:
Would it make sense to add text here that's representative of the lane operations that extract and replace single lanes instead of multiple values?

Reply:
Hm, there is no such breakdown in the intro sentence of the other changelog sections either. The details are in the bullet list, this is just for the general direction.

Reply:
Ack.